packages/sn-controls-react/src/fieldcontrols/Avatar/AvatarPicker.tsx

```diff
@@ -39,17 +39,15 @@ const pickerItemOptions: ODataParams<Folder> = {
  * Represents an avatar picker component
  */
 export function AvatarPicker(props: AvatarPickerProps) {
-  const { items, selectedItem, setSelectedItem, path, navigateTo, reload, isLoading, error } = useListPicker<
-    GenericContent
-  >({
+  const { items, selectedItem, setSelectedItem, path, navigateTo, reload, isLoading, error } = useListPicker<User>({
```

Comment:
Avatar picker is totally empty (no list, no upload button) on admin-ui
![image](https://user-images.githubusercontent.com/6968860/61532827-610bf500-aa2b-11e9-9f02-a24894f629fb.png)

Reply:
fixed. now it falls back to the avatar url path when no upload folder path or selection root is available.